backend/account/query.py

```diff
@@ -4,35 +4,37 @@
 Query functions for account related operations.
 """
 
-import datetime
 from django.core.exceptions import ObjectDoesNotExist
 from .models import User
 
+
 def get_or_create_user(payload: dict) -> User:
     try:
-        return User.objects.get(google_id=payload['google_id'])
+        return User.objects.get(google_id=payload["sub"])
     except ObjectDoesNotExist:
-        # TODO: Replace these dummy values with values populated using OAuth (#26)
         return User.objects.create(
             username=User.objects.all().count(),
-            google_id=payload['google_id'],
-            image_url='https://i.redd.it/9m2wxxxoljk91.jpg',
-            country='The Canada',
-            city='Middle of nowhere',
-            birthday=datetime.date(1970, 1, 1),
-            field_of_work="Wouldn't you like to know",
+            google_id=payload["sub"],
+            email=payload["email"] if payload.get("email") else "No Email Found",
+            image_url=payload["picture"] if payload.get("picture") else None,
+            first_name=payload["given_name"]
+            if payload.get("given_name")
+            else "No First Name Found",
+            last_name=payload["family_name"]
+            if payload.get("family_name")
+            else "No Last Name Found",
```

Comment:
Can we default to null or empty string instead of this? My google account has no last name and it sets my name up as "Speuce No Last Name Found" :P

Reply:
I believe `AbstractUser` requires non-null, but empty string is a good idea ya, could do that for all of them

Reply:
So is this an `if payload.get("family_name") else "..."` check then?

Reply:
Yeah so the line will be `last_name = payload.get("family_name") if payload.get("family_name") else ""`
Not sure if there's a more concise way of writing that or not

Reply:
changed last name to empty. Leaving email as-is